apps/vite/README.md

```diff
@@ -0,0 +1,6 @@
+# Example
```

Comment:
```suggestion
# Vite

A package for Volto. (I'm sure you can come up with something better.)

- What does this do?
- Who is it for?
- Add link to Vite site.

## Usage
```

Reply:
Not a package, a PoC. It's intended for devs interested in the Plone's headless CMS story or in doing things outside Volto (as all the other things in apps folder)